driver/TrEE/TrEE/FtpmService.c

```diff
@@ -25,8 +25,8 @@
 
 
 TEEC_UUID TaUuid = { 
-    SWAP_B32(0x53BAB89C), SWAP_B16(0xB864), SWAP_B16(0x4D7E),
-    {0xAC, 0xBC, 0x33, 0xC0, 0x7A, 0x9C, 0x1B, 0x8D}
+    SWAP_B32(0xBC50D971), SWAP_B16(0xD4C9), SWAP_B16(0x42C4),

```

Comment:
Since this changes the FtpmService TA GUID, and the iMX8 binaries haven't been updated, please create a task to track fixing that.

Reply:
Right, will do. I'll try to get that updated early next week.